From 5a9d1815868ff60871c6e5e56fe4ff1a260ea9ae Mon Sep 17 00:00:00 2001
From: Your Name <you@example.com>
Date: Tue, 17 Dec 2024 23:47:06 +0800
Subject: [PATCH 1/6] io_re_frida_server

---
 server/server.vala | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/server/server.vala b/server/server.vala
index 525c145e..7a6ec95a 100644
--- a/server/server.vala
+++ b/server/server.vala
@@ -1,7 +1,7 @@
 namespace Frida.Server {
 	private static Application application;
 
-	private const string DEFAULT_DIRECTORY = "re.frida.server";
+	private const string DEFAULT_DIRECTORY = "myserver";
 	private static bool output_version = false;
 	private static string? listen_address = null;
 	private static string? certpath = null;
-- 
2.24.1

