From 6cc07b51c2ad868357b60d45101028339abbf3f8 Mon Sep 17 00:00:00 2001
From: Your Name <you@example.com>
Date: Wed, 18 Dec 2024 01:25:30 +0800
Subject: [PATCH 6/6] protocol_unexpected_command

---
 src/droidy/droidy-client.vala | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/droidy/droidy-client.vala b/src/droidy/droidy-client.vala
index ddc56ccc..8b9e9c4c 100644
--- a/src/droidy/droidy-client.vala
+++ b/src/droidy/droidy-client.vala
@@ -1015,7 +1015,7 @@ namespace Frida.Droidy {
 						case "OPEN":
 						case "CLSE":
 						case "WRTE":
-							throw new Error.PROTOCOL ("Unexpected command");
+							break;
 
 						default:
 							var length = parse_length (command_or_length);
-- 
2.24.1

